src/rustdoc_gen.rs

```diff
@@ -90,10 +90,40 @@ impl<'a> CrateSource<'a> {
     }
 }
 
+#[derive(Debug, Clone)]
+pub(crate) struct FeatureSet {
+    pub(crate) default_features: bool,
+    pub(crate) features: Vec<String>,
+}
+
+#[allow(dead_code)]
+#[derive(Debug, Clone)]
+pub(crate) enum FeatureBehaviour {
+    AllFeatures,
+    SpecifiedFeatues(FeatureSet),
```

Comment:
Now that I think about it, making it an enum with named fields would be shorter and possibly more readable.

Reply:
Done in 44bde8abb45596dc39126788812e1bbd13727f07.